grails-app/domain/org/pih/warehouse/inventory/CycleCountRequest.groovy

```diff
@@ -0,0 +1,61 @@
+package org.pih.warehouse.inventory
+
+import org.pih.warehouse.auth.AuthService
+import org.pih.warehouse.core.Location
+import org.pih.warehouse.core.User
+import org.pih.warehouse.product.Product
+
+class CycleCountRequest {
+
+    String id
+
+    Location facility
+
+    Product product
+
+    CycleCountStatus status
```

Comment:
I was assuming this would be a `CycleCountRequestStatus` enum. Won't a CycleCountRequest and CycleCount have separate statuses?

Reply:
Yeah, if so, I can rename it to `CycleCountRequestStatus`.